### PR TITLE
[PPD-227] iuv generator integration test

### PR DESCRIPTION
**Pull request by alessio-acitelli**
#### List of Changes

- added cucumber integration test

#### Motivation and Context

- integration test for the iuv generator service

#### How Has This Been Tested?

local run

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by jacopocarlini — APPROVED** [review]
✅

File: integration-test/step_definitions/support/iuv_steps.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+const {generateIUV} = require("./iuv_client");
+const {randomOrg} = require("./common");
+
+let responseToCheck;
+let iuv;
+let idOrg;
+
+// Given
+Given('IUV Generator Function running', async function () {
+	//TODO health check call
```
Comment (jacopocarlini) [inline integration-test/step_definitions/support/iuv_steps.js]: [use this endpoint](https://github.com/pagopa/pagopa-iuv-generator/blob/main/src/main/java/it/gov/pagopa/iuvgenerator/Info.java#L24)

**Review by jacopocarlini — CHANGES_REQUESTED** [review]
ok, but implement health check call

File: integration-test/step_definitions/support/iuv_steps.js (hunk 1 above)
Comment (alessio-acitelli) [reply integration-test/step_definitions/support/iuv_steps.js]: done

File: integration-test/features/iuv.feature [hunk 2]
```diff
@@ -9,7 +13,7 @@ Feature: Create a valid IUV
     Then the organization gets the status code 201
     And the length of the iuv is 17 digits
   
-  #Case KO: 400 BAD REQUEST  
+  # Case KO: 400 BAD REQUEST  
   Scenario: An organization requests the generation of an IUV with bad body request
     Given an organization
     When the organization ask for a valid IUV with incorrect body request
```
Comment (jacopocarlini) [inline integration-test/features/iuv.feature]:
```suggestion
    When the organization asks for a valid IUV with incorrect body request
```

File: integration-test/step_definitions/support/iuv_steps.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	responseToCheck = await healthCheckInfo();
+	assert.strictEqual(responseToCheck.status, 200);
+});
+
+Given('an organization', async function() {
+	idOrg = randomOrg();
+});
+
+// When
+When('the organization ask for a valid IUV', async function() {
```
Comment (jacopocarlini) [inline integration-test/step_definitions/support/iuv_steps.js]:
```suggestion
When('the organization asks for a valid IUV', async function() {
```

**Review by jacopocarlini — APPROVED** [review]
great!

**Pull request merged**